package.json

```diff
@@ -116,6 +117,6 @@
     "which": "^4.0.0"
   },
   "peerDependencies": {
-    "testplane": ">8.20.0"
+    "testplane": ">=8.39.0"
```

Comment:
Maybe it looks like major change, but without new version of testplane `repl` mode doesn't work anymore. So I decided to update the dependencies.